projects/netgrif-components-core/src/lib/data-fields/number-field/currency-number-field/abstract-currency-number-field.component.ts

```diff
@@ -24,7 +25,7 @@ export abstract class AbstractCurrencyNumberFieldComponent extends AbstractNumbe
         this.numberField.valueChanges().subscribe(value => {
             if (value !== undefined) {
                 if (this.fieldType === this.TEXT_TYPE) {
-                    this.transformedValue = this.transformCurrency(value.toString());
+                    this.transformedValue = this.transformCurrency(value.toString()) + this.WHITESPACE;
```

Comment:
So junky solution ... 😄 
Please create an improvement issue to refactor change detection so similar problems do not arise in the future.

Reply:
[NAE-1727](https://netgrif.atlassian.net/browse/NAE-1727)